package.json

```diff
@@ -23,8 +23,9 @@
     "update-translations": "fetch-translations 1b5qAamjhmSNiME3matBINaeyz8TvoRWVB6wU5HvKy4g -l en,fi,sv -o src/common/translation/i18n",
     "test": "TZ=UTC react-scripts test --transformIgnorePatterns \"node_modules/(?!(react-helsinki-headless-cms|react-helsinki-headless-cms/apollo))/\"",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
-    "test:browser": "testcafe \"chrome --window-size='1920,1080'\" browser-tests/ --live --lang=fi-FI",
-    "test:browser:ci": "testcafe \"chrome:headless --disable-gpu --window-size='1920,1080'\" browser-tests/ -s takeOnFails=true --lang=fi-FI"
+    "test:browser": "testcafe \"chrome --window-size='1920,1080'\" browser-tests/ --live --dev --lang=fi-FI --skip-js-errors",
```

Comment:
why `--skip-js-errors`? Sometimes errors are important information. 
If you use something like this https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/testcafeClientErrorHandler.js
You can use that to filter out specific known errors that are not important like ResizeObserver

Reply:
You have to refer to that handler from testcafe settings: https://github.com/City-of-Helsinki/yjdh/blob/1abe148649a7699941ce262b094ac36a5c61e3ff/frontend/.testcaferc.base.js#L19

Reply:
I don't know is this real problem (and how to fix it) or should this just skip in browser tests.
I created a bug report  about this error
https://helsinkisolutionoffice.atlassian.net/browse/KK-955

Reply:
Only this specific error note is skipped, details are documented on KK-955